src/invite.tsx

```diff
@@ -21,9 +21,20 @@ const cn = bemClassName('invite-main');
 export interface Properties {
   stage: RegistrationStage;
   shouldRender: boolean;
+
+  registerWithEmail: () => Action<string>;
+  registerWithWallet: () => Action<string>;
+}
+
+export interface State {
+  selectedOption: string;
 }
 
 export class Container extends React.Component<Properties> {
+  state: State = {
+    selectedOption: 'web3',
```

Comment:
Not sure if we even need this since we have the Stages that we could key off of? Not worth changing unless it's a 5 minute swap.